pkg/operator/ceph/object/objectstore.go

```diff
@@ -736,7 +737,13 @@ func CreatePools(context *Context, clusterSpec *cephv1.ClusterSpec, metadataPool
 	}
 
 	// get the default PG count for rgw metadata pools
-	metadataPoolPGs, err := config.GetMonStore(context.Context, context.clusterInfo).Get("mon.", "rgw_rados_pool_pg_num_min")
+	var err error
+	var metadataPoolPGs string
```

Comment:
How about a unit test that tests the different settings for different ceph versions?

Reply:
can you please provide examples for fetching these details?